conf/models.yml

```diff
@@ -57,12 +50,9 @@ model: &mlp
         dimensions: [100]
     opt_alg_name: adam
     batch_size: 100
-    lr: 5e-3
+    lr: !!float 5e-3
```

Comment:
[yaml documentation](https://pyyaml.org/wiki/PyYAMLDocumentation#yaml-tags-and-python-types)

is there a reason we want to specify the float type here, but not the other types (int, str, etc.)?

Reply:
the value here defaults to a string (because of the "e"), so we cast to a float for torch's optimizers